kitchen.yml

```diff
@@ -134,8 +116,21 @@ verifier:
   # cli, documentation, html, progress, json, json-min, json-rspec, junit
   reporter:
     - cli
-  inspec_tests:
-    - path: test/integration/default
 
 suites:
   - name: default
+    provisioner:
+      state_top:
+        base:
+          '*':
+            - openvpn.config
```

Comment:
So we don't test the metastate "openvpn"? only "openvpn.config"? There must have a reason. Anyway it was already like that so no impact on your PR.

Reply:
Yes, noticed a few of the formulas doing this.  It's much easier to get an overview when all of the configuration elements are in the same file.  So we'll be able to investigate this a further as we make progress with all of this.